# AttEuler block
# ROS message header
std_msgs/Header header

# SBF block header including time header
BlockHeader  block_header   

uint8        nr_sv
uint8        error        
uint16       mode
float32      heading
float32      pitch
float32      roll
float32      pitch_dot
float32      roll_dot
float32      heading_dot
